lib/kafo/hook_context.rb

```diff
@@ -3,22 +3,23 @@
 
 module Kafo
   class HookContext < BaseContext
-    attr_reader :kafo
+    attr_reader :kafo, :logger
 
-    def self.execute(kafo, &hook)
-      new(kafo).instance_eval(&hook)
+    def self.execute(kafo, logger, &hook)
+      new(kafo, logger).instance_eval(&hook)
     end
 
-    def initialize(kafo)
+    def initialize(kafo, logger)
```

Comment:
This appears to be breaking a lot of tests.

For example in `test/kafo/hook_context_test.rb:6`:

`   let(:context) { HookContext.new(kafo) }`

This now expects a 2nd argument